dart/5.Command_line_app/6_pseudo_code_of_the_game.md

```diff
@@ -0,0 +1,24 @@
+# _Pseudo code_ for the game
+
+As we have been doing, let's write what we need to do in a colloquial way according to the requirements:
+
+```dart
+void main() {
+    print('Hello world!');
+
+
+    //  Pseudo code
```

Comment:
Is that possible and correct to align this like in the original file so we show different levels?

Reply:
done